docs/man/containerd-config.toml.5.md

```diff
@@ -13,15 +14,27 @@ does not exist at the appropriate location or is not provided via the
 **--config** option containerd uses its default configuration settings, which
 can be displayed with the **containerd config(1)** command.
 
-## DESCRIPTION
+## Description
 
-The TOML file used to configure the containerd daemon settings has a short
+The **config.toml** file used to configure the containerd daemon settings has a short
 list of global settings followed by a series of sections for specific areas
 of daemon configuration. There is also a section for **plugins** that allows
 each containerd plugin to have an area for plugin-specific configuration and
 settings.
 
-## FORMAT
+## Config versions
```

Comment:
when changing the names of these sections have to review if there are any section links from any doc

Reply:
Is the section name case-sensitive for corresponding section links?

Reply:
Looks like it's not case sensitive as I tried access the ``crictl.md`` section `Download and Inspect a Container Image
` using `download and inspect a container image` and it was working as expected. 


https://github.com/matteopulcini6/containerd/blob/main/docs/cri/crictl.md#download-and-inspect-a-container-image